boards/riscv/longan_nano/support/openocd.cfg

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2021 Tokita, Hiroshi <tokita.hiroshi@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Hi @soburi ,

I made some tests and works better this way:
1) create a new file gd32vf103.cfg and add content of https://gist.github.com/elfmimi/1deb9c94b0f0900ae8a9df740b62bcd6
2) Replace all content with this:
```
# Copyright (c) 2021 Tokita, Hiroshi <tokita.hiroshi@gmail.com>
# SPDX-License-Identifier: Apache-2.0

interface ftdi
#ftdi_device_desc "Dual RS232"
#ftdi_device_desc "Sipeed-Debug"
ftdi_device_desc "JTAG Debugger"

ftdi_vid_pid 0x0403 0x6010
#ftdi_channel 0

transport select jtag
ftdi_layout_init 0x0008 0x001b
ftdi_layout_signal nSRST -oe 0x0020 -data 0x0020

source [find gd32vf103.cfg]
```

People can select between programmers and everything works better.

Reply:
Hi @elfmimi,
 Sorry for the sudden mention.

 We want to commit your great patch https://gist.github.com/elfmimi/1deb9c94b0f0900ae8a9df740b62bcd6 for resetting the GD32VF103 to zephyr.

 The patch is on your gist.
 Could you please make the patch as a git commit?
 (Because I shouldn't copy and commit it without your permission. And I think your contribution should be recorded and honored.)

I'm sorry for the additional request.
Would you follow two zephyr conventions, please?
- Add the SPDX Identifier at the beginning of the file
- Sign-off the commit message

When the patch push to GitHub, I will cherry-pick it and commit it in this PR.

@nandojve Is it the correct way to apply the @elfmimi's patch?

Reply:
Hi @soburi , yes, without @elfmimi authorization it is not possible apply the solution. Ask for authorization is part of process.

> When the patch push to GitHub, I will cherry-pick it and commit it in this PR.
Yes, best solution.

I think it is better have 2 additional commits: one only to create **gd32vf103.cfg** file and another to replace content from `openocd.cfg`. The @elfmimi content must be send isolated and as-is to keep their original rights, since he is the `Author`. This special commit will require both `Signed-off-by` to submit patch to Zephyr repo to state that he participated in the review process (this thread). After that commit, the content can be changed without problems. This keeps clear what each author is doing avoiding confusions.

@carlescufi , feel free to advise since this is not a daily base case.

Reply:
Modify  openocd.cfg to able to  select programmers.
At this time, We not receive reply from @elfmimi, gd32vf103.cfg will update future.